examples/flutter_view/lib/main.dart

```diff
@@ -0,0 +1,85 @@
+import 'dart:async';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() {
+  runApp(new FlutterView());
+}
+
+class FlutterView extends StatelessWidget {
+
+  @override
+  Widget build(BuildContext context) {
+    return new MaterialApp(
```

Comment:
I wonder if it would be move realistic in this sample if we had the main chrome (title bar) be owned by the native side, then only have Flutter render contents inside it?